2022q3/bhyve-debug-server-enhancements.adoc

```diff
@@ -0,0 +1,25 @@
+=== bhyve debug server enhancements
+
+Links: +
+link: https://wiki.freebsd.org/SummerOfCode2022Projects/BhyveDebugServerEnhancements [Wiki project page]
+link: https://reviews.freebsd.org/D35826 [Differential]
```

Comment:
I think there should be no space between the URL and the link text in brackets.

Reply:
Fixed, thank you for pointing it out.